.github/dependabot.yml

```diff
@@ -3,7 +3,13 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/editoast/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+    groups:
+      opentelemetry:
+        patterns:
+          - "opentelemetry*"
+          - "tracing-opentelemetry"
+          - "axum-tracing-opentelemetry"
```

Comment:
```suggestion
    groups:
      opentelemetry:
        patterns:
          - "*opentelemetry*"
```

Reply:
I intentionally did not do that in the idea that whatever is prefixed by `opentelemetry` is owned by Opentelemetry developers, and the others are not, and therefore, let's cherry pick... but hey, not sure it's that useful in the end so done in [7206569](https://github.com/OpenRailAssociation/osrd/pull/8998/commits/720656930dc9bb32e36d4c6648f7f7b796be46d5) (and also in [1062409](https://github.com/OpenRailAssociation/osrd/pull/8998/commits/10624093d8dbb06963619a9f66dd2f4baba1c61c) for `gateway`).